### PR TITLE
Add ability to get filtered next event from changelog

### PR DESCRIPTION
**Pull request by janderson-seg**
Added new method to Iterator, similar to Next but adds the ability to only return events for family/table user cares about. 

Also cut out some verbose logging from the changelog.

Testing completed successfully via unit tests

### CODE REVIEW

File: pkg/event/iterator.go [hunk 1]
```diff
@@ -69,7 +70,19 @@ func (i *Iterator) Next(ctx context.Context) (event Event, err error) {
 			return event, ErrOutOfSync
 		}
 	}
-	return
+	return event, err
+}
+
+// NextForFamilyTable blocks and returns the next event that matches the specified family and table
+func (i *Iterator) NextForFamilyTable(ctx context.Context, family, table string) (event Event, err error) {
```
Comment (wavetylor) [inline pkg/event/iterator.go:77]: so if int-consumer wanted to use this, it would be to open an iterator for every table it wanted to track?

Comment (janderson-seg) [reply pkg/event/iterator.go:77]: Yes, do you see any issues with that?

Comment (janderson-seg) [reply pkg/event/iterator.go:77]: Correct, the use of `Next` and `NextForFamilyTable` together would not make sense and would not be the intended usage model. Perhaps to make this clearer a separate struct would be better. Will modify and update shortly.

**Review by wavetylor — APPROVED** [review]

File: pkg/event/iterator.go (hunk 1 above)
Comment (wavetylor) [reply pkg/event/iterator.go]: looks great, thank you for doing that

**Pull request merged**